### PR TITLE
Redefine window.event after raise native click in IE11 (close #226)

### LINKED ISSUE #226: Window.event becomes empty when a click event handler triggers the click event on a different element in IE11
T304784

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @churkin

### CODE REVIEW

File: src/client/sandbox/event/simulator.js [hunk 1]
```diff
@@ -222,6 +223,15 @@ export default class EventSimulator {
                 });
             }
         }
+
+        // NOTE: Window.event becomes empty when a click event handler
+        // triggers the click event on a different element in IE1(GH-226)
```
Comment (churkin) [inline src/client/sandbox/event/simulator.js]: IE1

Comment (churkin) [inline src/client/sandbox/event/simulator.js]: @VasilyStrelyaev Check pls

File: src/client/sandbox/event/simulator.js [hunk 2]
```diff
@@ -376,115 +386,124 @@ export default class EventSimulator {
...
+            var isElementInIframe = domUtils.isElementInIframe(el);
+            var iframe            = isElementInIframe ? domUtils.getIframeByElement(el) : null;
+            var curWindow         = iframe ? iframe.contentWindow : window;
+
+            if (browserUtils.version < 11) {
+                args = args || { type: ev.type };
+
+                var returnValue    = true;
+                var curWindowEvent = null; // NOTE: B254199
+                var MSPointerRegExp = /MSPointer(Down|Up|Move|Over|Out)/;
```
Comment (inikulin) [inline src/client/sandbox/event/simulator.js]: Move this RegExp to constants. We used `_RE` postfix for constants now

**Comment by inikulin** [comment]
\r-

File: src/client/sandbox/event/simulator.js (hunk 1 above)
Comment (VasilyStrelyaev) [inline src/client/sandbox/event/simulator.js]:
```
        // NOTE: In IE11, window.event becomes empty when the click event handler
        // triggers the click event for a different element. (GH-226)
```
Don't ask me how do I know it's IE11. I do know.

**Comment by churkin** [comment]
\r-

File: src/client/sandbox/event/simulator.js [hunk 3]
```diff
@@ -222,6 +224,15 @@ export default class EventSimulator {
                 });
             }
         }
+
+        // NOTE: Window.event becomes empty when a click event handler
+        // triggers the click event on a different element in IE11(GH-226).
```
Comment (VasilyStrelyaev) [inline src/client/sandbox/event/simulator.js]: My comment got lost, so again:
```
        // NOTE: In IE11, window.event becomes empty when the click event handler 
        // triggers the click event for a different element.
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by helen-dikareva** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**